### ISSUE TITLE
JSONDecodeError when search result is empty

### ISSUE THREAD

**Issue by svenvanderburg** [fair-workflows/nanopub#111]
```python
results = client.find_nanopubs_with_text('some random string that is not in any of the nanopublications')
print(next(results))
```
```python
---------------------------------------------------------------------------
JSONDecodeError                           Traceback (most recent call last)
<ipython-input-168-bab0d5cfb459> in <module>
      1 results = client.find_nanopubs_with_text('some random string that is not in any of the nanopublications')
----> 2 print(next(results))

~/projects/fair-workflows/nanopub/nanopub/client.py in _search(self, endpoint, params)
    246                 r = self._query_grlc(params, endpoint, grlc_url)
    247                 r.raise_for_status()
--> 248             results = r.json()
    249             bindings = results['results']['bindings']
    250             if not bindings:

~/opt/anaconda3/envs/fair3.6/lib/python3.6/site-packages/requests/models.py in json(self, **kwargs)
    896                     # used.
    897                     pass
--> 898         return complexjson.loads(self.text, **kwargs)
    899 
    900     @property

~/opt/anaconda3/envs/fair3.6/lib/python3.6/json/__init__.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    352             parse_int is None and parse_float is None and
    353             parse_constant is None and object_pairs_hook is None and not kw):
--> 354         return _default_decoder.decode(s)
    355     if cls is None:
    356         cls = JSONDecoder

~/opt/anaconda3/envs/fair3.6/lib/python3.6/json/decoder.py in decode(self, s, _w)
    337 
    338         """
--> 339         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    340         end = _w(s, end).end()
    341         if end != len(s):

~/opt/anaconda3/envs/fair3.6/lib/python3.6/json/decoder.py in raw_decode(self, s, idx)
    355             obj, end = self.scan_once(s, idx)
    356         except StopIteration as err:
--> 357             raise JSONDecodeError("Expecting value", s, err.value) from None
    358         return obj, end

JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by raar1** [comment]
Yeah, I think the problem here is that it is receving HTML instead of JSON, since virtuoso has given an error (it doesn't like the spaces in the free form text). I've tried to handle this a little more gracefully in PR #117

**Comment by raar1** [comment]
Fixed in PR #117
